skyagi-web/src/lib/stores.ts

```diff
@@ -0,0 +1,3 @@
+import { writable } from "svelte/store";
+
+export const isAgentFormEditing = writable(false);
```

Comment:
Since you put everything in handleSubmit, that's why you need this global state isAgentFormEditing to keep track, which is also unnecessary. I can refactor later.

Reply:
I also use the this state to conditionally show the `cancel` button:
```
{#if $isAgentFormEditing}
	<Button type="button" class="" color="light" on:click={() => isAgentFormEditing.set(false)}
		>Cancel</Button
	>
{/if}
```
So I think we still need it.